mojblocks.php

```diff
@@ -547,6 +569,25 @@ function mojblocks_extend_wp_api($data, $post, $context) {
         $data->data['featured_image_url'] = "";
     }
 
+    // Add summary to API
+    $summary = get_post_meta( get_the_ID(), 'post_summary', TRUE); // get the value from the meta field
```

Comment:
Worth a discussion as we talking about adding something into hale that removes the need for this

Reply:
Done